Fix C++ Octal Notation with old llvm-20 on el9, el10.1 and f42

../../components/named_mojo_ipc_server/named_mojo_server_endpoint_connector_linux.cc:119:46: error: invalid suffix 'o666' on integer constant
  119 |     if (chmod(options_.server_name.c_str(), 0o666) != 0) {
      |                                              ^
1 error generated.

diff -up chromium-146.0.7680.65/components/named_mojo_ipc_server/named_mojo_server_endpoint_connector_linux.cc.than chromium-146.0.7680.65/components/named_mojo_ipc_server/named_mojo_server_endpoint_connector_linux.cc
--- chromium-146.0.7680.65/components/named_mojo_ipc_server/named_mojo_server_endpoint_connector_linux.cc.than	2026-03-10 14:32:08.383467214 +0100
+++ chromium-146.0.7680.65/components/named_mojo_ipc_server/named_mojo_server_endpoint_connector_linux.cc	2026-03-10 14:32:32.286698674 +0100
@@ -116,7 +116,7 @@ bool NamedMojoServerEndpointConnectorLin
   if (!options_.require_same_peer_user) {
     // Allow any user to write to the UDS. fchmod doesn't work after bind(), so
     // we need to call chmod on the socket filename, which is the server name.
-    if (chmod(options_.server_name.c_str(), 0o666) != 0) {
+    if (chmod(options_.server_name.c_str(), 0666) != 0) {
       PLOG(ERROR) << "chmod failed";
       return false;
     }
